docs/examples/errorprone/build.gradle

```diff
@@ -0,0 +1,32 @@
+///////////////////////////////////////////////////////////////////////////
+/// Checker Framework pluggable type-checking and Error Prone example
+///
+
+plugins {
+    id 'java'
+    id 'net.ltgt.errorprone' version '0.8.1'
+    // Checker Framework pluggable type-checking
+    id 'org.checkerframework' version '0.3.28' apply false
```

Comment:
@smillst 
The `apply false` doesn't seem to solve the problem here. Only the Error Prone checks are executed and no Checker Framework errors are produced.
Do you have another idea?

Reply:
I changed the expected Error Prone error to a warning, and now the ErrorProne warning and the Nullness Checker  error are raised.  The Nullness Checker won't run if there is an ErrorProne error. (I have no idea why there are not issue with the older versions of the javacutil and dataflow jars though.)

Reply:
Ah, that means that they run as two separate compilations, not as one that does both checks at once.
I think that behavior is good enough for now and https://github.com/kelloggm/checkerframework-gradle-plugin/issues/39 can look into whether this could be done more efficiently.

Reply:
I think this only works because checker.jar is first on the processorpath.  For example:

```
javac   -XDcompilePolicy=simple   \
-processorpath $CHECKERFRAMEWORK/checker/dist/checker.jar:error_prone_core-2.3.2-with-dependencies.jar:dataflow-2.5.7.jar:javacutil-2.5.7.jar:jFormatString-3.0.0.jar   \
'-Xplugin:ErrorProne -XepDisableAllChecks -Xep:CollectionIncompatibleType:WARN'  $CHECKERFRAMEWORK/docs/examples/errorprone/src/main/java/Demo.java  -processor org.checkerframework.checker.nullness.NullnessChecker -cp $CHECKERFRAMEWORK/checker/dist/checker-qual.jar   -J-Xbootclasspath/p:javac-9+181-r4173-1.jar
```

Works, but

```
javac   -XDcompilePolicy=simple   \
-processorpath error_prone_core-2.3.2-with-dependencies.jar:dataflow-2.5.7.jar:javacutil-2.5.7.jar:jFormatString-3.0.0.jar:$CHECKERFRAMEWORK/checker/dist/checker.jar   \
'-Xplugin:ErrorProne -XepDisableAllChecks -Xep:CollectionIncompatibleType:WARN'  $CHECKERFRAMEWORK/docs/examples/errorprone/src/main/java/Demo.java  -processor org.checkerframework.checker.nullness.NullnessChecker -cp $CHECKERFRAMEWORK/checker/dist/checker-qual.jar   -J-Xbootclasspath/p:javac-9+181-r4173-1.jar
```

throws:
```
An annotation processor threw an uncaught exception.
Consult the following stack trace for details.
java.lang.NoSuchMethodError: org.checkerframework.javacutil.PluginUtil.getJreVersion()I
	at org.checkerframework.framework.source.SourceChecker.init(SourceChecker.java:457)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment$ProcessorState.<init>(JavacProcessingEnvironment.java:675)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment$DiscoveredProcessors$ProcessorStateIterator.next(JavacProcessingEnvironment.java:774)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment.discoverAndRunProcs(JavacProcessingEnvironment.java:869)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment.access$2200(JavacProcessingEnvironment.java:108)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment$Round.run(JavacProcessingEnvironment.java:1206)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment.doProcessing(JavacProcessingEnvironment.java:1315)
	at com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1246)
	at com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:922)
	at com.sun.tools.javac.main.Main.compile(Main.java:302)
	at com.sun.tools.javac.main.Main.compile(Main.java:162)
	at com.sun.tools.javac.Main.compile(Main.java:57)
```